src/main/java/com/agency04/sbss/pizza/rest/DeliveryRestController.java

```diff
@@ -1,44 +1,82 @@
 package com.agency04.sbss.pizza.rest;
 
-import com.agency04.sbss.pizza.model.DeliveryOrderForm;
-import com.agency04.sbss.pizza.service.PizzaDeliveryService;
+
+import com.agency04.sbss.pizza.DAO.DeliveryRepository;
+import com.agency04.sbss.pizza.DAO.PizzaOrderRepository;
+import com.agency04.sbss.pizza.model.Customer;
+import com.agency04.sbss.pizza.model.Delivery;
+import com.agency04.sbss.pizza.model.PizzaOrder;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import javax.annotation.PostConstruct;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Date;
+import java.util.Optional;
 
 @RestController
 @RequestMapping("/api/delivery")
 public class DeliveryRestController {
 
     @Autowired
-    private PizzaDeliveryService deliveryService;
-    private List<DeliveryOrderForm> currentOrders;
-
+    private DeliveryRepository deliveryRepository;
+    @Autowired
+    private PizzaOrderRepository pizzaOrderRepository;
 
-    @PostConstruct
-    public void loadData() {
-        currentOrders = new ArrayList<>();
+    @Autowired
+    public DeliveryRestController(DeliveryRepository theDeliveryRepository, PizzaOrderRepository pizzaOrderRepository){
+        deliveryRepository = theDeliveryRepository;
+        this.pizzaOrderRepository = pizzaOrderRepository;
     }
 
     @PostMapping("/order")
-    public ResponseEntity orderDetails (@RequestBody DeliveryOrderForm deliveryOrderForm){
-        currentOrders.add(new DeliveryOrderForm(deliveryOrderForm));
-        DeliveryOrderForm check = deliveryService.orderPizza(deliveryOrderForm);
-        if(check == deliveryOrderForm){
-            return ResponseEntity.ok(HttpStatus.CREATED).status(201).build();
+    public PizzaOrder orderDetails (@RequestBody PizzaOrder pizzaOrder){
+        pizzaOrderRepository.save(pizzaOrder);
+        deliveryRepository.save(new Delivery(0, new Date()));
```

Comment:
Zašto se postavlja vrijednost ID-a 0? Također koristiti LocalDate i LocalDateTime klase umjesto Date.